wordpress-mobile/WordPress-Android
# Issue/view more comments

This PR adds the new View all screen for comments.

To test:
1. Go to Stats
2. Scroll to the Comments block
3. Tap the View more button
4. Notice the new View all screen gets loaded with additional authors and posts entries

Note: The #9164 should be merged before this one and the base branch should be updated.

![image](https://user-images.githubusercontent.com/1522856/52417940-41bc8380-2aed-11e9-8402-a90670f282da.png)
